go.mod

```diff
@@ -44,14 +44,15 @@ require (
 	golang.org/x/term v0.5.0
 	golang.org/x/time v0.3.0
 	google.golang.org/grpc v1.53.0
-	google.golang.org/protobuf v1.28.1
+	google.golang.org/protobuf v1.28.2-0.20220831092852-f930b1dc76e8
```

Comment:
mmm...this is a not released version. Not optimal, now that we need to use protobuf for other things I hope it will not create issues.

Reply:
I didn't manually update this, it came from the github.com/jhump/protoreflect update, see [this](https://github.com/jhump/protoreflect/blob/bc601aed79cb23f3a46dd0a2817d139551f94c28/go.mod#L11) :man_shrugging: